sites/fast-website/src/public/index.html

```diff
@@ -6,127 +6,114 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider id="body-provider" class="body-provider"
+    <fast-design-system-provider
+      class="body-provider"
       use-defaults
-      background-color="#1F1F1F"
-      accent-base-color="#F33378"
+      background-color="#181818"
+      accent-base-color="#DA1A5F"
+      type-ramp-base-font-size="16px"
```

Comment:
What's the reason for deviating from the default values for type ramp here? Should these changes just be made in the design system provider itself?

Reply:
I think so. The marketing site should be a one to one of `fast-components`

Reply:
Based on the design, the majority of body text-esque parts are at 16px, including the section button links:
![image](https://user-images.githubusercontent.com/863023/82956271-4d8fbb80-9f65-11ea-99d7-07f893babd7f.png)

However, text in cards is 14px, same with header navigation links.

I also think 16px as a base makes sense since it's usually the brower's default font size, and cleanly divides into the default design-unit value of 4px.

Reply:
Yea I don't have a strong opinion on what it should but - just noting that we're out of alignment. Looping in @bheston since he did most of the typeramp work.

Reply:
We can take this as a followup